explain/setup/index.mdx

```diff
@@ -34,7 +34,7 @@ Platform         | Log-based EXPLAIN |    auto_explain   |
 -----------------|-------------------|-------------------|
 Amazon RDS       | Yes               | Yes (Recommended) |
 Azure Database   | Yes               | No                |
```

Comment:
Not urgent, but I just realized we haven't updated the Azure docs to reflect that Azure also supports auto_explain now. We can discuss separately how we want to prioritize that.

Reply:
Ah, I didn't even know Azure had auto_explain, too. But yeah, let's do that separately to avoid holding this up.